Changes to Marketing Lead JD (#96)
OpenZeppelin/openzeppelin.com: _jobs/Marketing-Lead.md

  <br/>
  <ul>
    <li><p><b>Driving awareness, generating leads, accelerating pipeline and growing community</b> through a cohesive marketing strategy that binds all OpenZeppelin’s offerings (Open Source tools, Product, Services). Programs can include outbound email and direct mail campaigns, upsell & cross-sell campaigns, webinars, partner activities, content marketing, social and online marketing, and OpenZeppelin and third-party events. </p></li>
    <li><p><b>Designing and executing an editorial plan for content marketing</b> that showcases OpenZeppelin’s offerings and benefits to our target audience. Identify topics and motivate the entire organization and external community members to create original content. This may include blog posts, research and reports, guides, case studies, presentations and sales material.</p></li>
    <li><p><b>Defining an event attendance strategy</b> by sourcing, planning, and coordinating meetups, hackathons and industry events around the world. Find speaking opportunities at conferences and define actions to drive lead generation and increase brand awareness and community engagement as a result of them.</p></li>

>     <li><p><b>Coordinating product launches, new releases and company announcements</b>. Partner with Product, Open Source and Services teams to craft messaging and create material.Define and execute distribution strategy through various channels, including managing relationships with journalists and influencers, and working with a PR agency.</p></li>

Reviewer: ```suggestion
    <li><p><b>Coordinating product launches, new releases and company announcements</b>. Partner with Product, Open Source and Services teams to craft messaging and create material. Define and execute distribution strategy through various channels, including managing relationships with journalists and influencers, and working with a PR agency.</p></li>
```

    <li><p><b>Defining, implementing, and managing the necessary toolset</b> to interact with our target audience and community.</p></li>
    <li><p><b>Tracking, measuring and communicating program results</b>. Report program results and future priorities with OpenZeppelin leadership team.</p></li>
    <li><p><b>Ensuring compliance of OpenZeppelin brand strategy</b> across all internal and external outlets and publications, including both copywriting and visual style.</p></li>
  </ul>
</div>
<div class="requirements">
  <h3 class="job-description-title">You have</h3>
  <ul>
    <li><p><b>3+ years of previous experience working on international marketing strategies in top technology or cybersecurity companies</b>. Specific experience with designing, executing, and tracking results of multiple lead generation programs including upsell & cross-sell campaigns, content marketing, email campaigns, and event management.</p></li>
    <li><p><b>Experience working in a technical industry and work environment with a strong B2B component</b>. Experience in the cybersecurity space and understanding of developer audiences and blockchain technology are big pluses.</p></li>

>     <li><p><b>A startup mentality with a bias to action</b> and the ability to flex in a fast-paced environment.</p></li>

Reviewer 2: ```suggestion
    <li><p><b>A startup mentality with a bias to action</b> and the ability to be flexible in a fast-paced environment.</p></li>
```
Likely this is what you meant:

flex: intransitive verb
 - flex one's muscles
 : to demonstrate one's strength
  __'an exaggerated need to flex his political muscles'__

(https://www.merriam-webster.com/dictionary/flex))

Reviewer 2: Looks like I arrived 30 seconds late @elopio 😂

    <li><p><b>Superb organization and project management skills</b>. Ability to work on and manage multiple complex projects with multiple stakeholders, at once.</p></li>
    <li><p><b>Excellent written and verbal communication skills</b>. Proven ability to communicate complex topics with simplicity and clarity, and to communicate powerful stories to drive demand. Proven ability to create, edit and promote written and visual content.</p></li>

>     <li><p><b>Ability to think critically about complex problems with an analytical mindset</b>,appropriately weigh many factors and reach a definitive conclusion.</p></li>

Reviewer 2: ```suggestion
    <li><p><b>Ability to think critically about complex problems with an analytical mindset</b>, appropriately weigh many factors and reach a definitive conclusion.</p></li>
```

Missing space
